RELEASE_NOTES.md

```diff
@@ -0,0 +1,10 @@
+# Unreleased
+
+- Umpire is MPI-aware (outputs rank information to logs and replays) when
+  configured with the option ENABLE_MPI=On
+
+- AllocationStrategies may be wrapped with mulptile extra layers - to "unwrap"
+  an Allocator to a specific strategy, the
+  umpire::util::unwrap_allocator method can be used. This will impact users who
+  have been using DynamicPool::coalesce - the cookbook recipe has been updated
+  accordingly.
```

Comment:
Can we provide a link to the cookbook recipe here?

Reply:
To the file, or the ReadTheDocs?

Reply:
Actually, I think it might be better to provide a link to the diff from the previous version of the recipe as shown in this PR.